api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8770,4 +8770,205 @@
 		<!-- visit_attribute_type has not null constraint on uuid -->
 		<!-- visit_type has not null constraint on uuid -->
 	</changeSet>
+	
+	<changeSet id="20151006-1530" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_set" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_set table
+		</comment>
+		<createTable tableName="order_set">
+			<column name="order_set_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="operator" type="varchar(50)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="name" type="varchar(255)">
+				<constraints nullable="false"/>
+			</column>
+			<column name="description" type="varchar(1000)"/>
+			<column name="creator" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="date_created" type="DATETIME">
+				<constraints nullable="false"/>
+			</column>
+			<column defaultValueBoolean="false" name="retired" type="BOOLEAN">
+				<constraints nullable="false"/>
+			</column>
+			<column name="retired_by" type="int"/>
+			<column name="date_retired" type="DATETIME"/>
+			<column name="retire_reason" type="varchar(255)"/>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="DATETIME"/>
+			<column name="uuid" type="char(38)">
+				<constraints unique="true"/>
+			</column>
+		</createTable>
+		<addForeignKeyConstraint constraintName="order_set_creator"
+								 baseTableName="order_set" baseColumnNames="creator"
+								 referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_set_retired_by"
+								 baseTableName="order_set" baseColumnNames="retired_by"
+								 referencedTableName="users" referencedColumnNames="user_id"/>
+		<addForeignKeyConstraint constraintName="order_set_changed_by"
+								 baseTableName="order_set" baseColumnNames="changed_by"
+								 referencedTableName="users" referencedColumnNames="user_id"/>
+	</changeSet>
+	<changeSet id="20151006-1537" author="bahmni">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_set_member" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_set_member table
+		</comment>
+		<createTable tableName="order_set_member">
+			<column name="order_set_member_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="order_type" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="order_template" type="text"/>
+			<column name="order_set_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="sort_weight" type="int">
```

Comment:
@lluismf – the `double` datatype is referring to the database attribute's datatype, not in Java.

@ys-achinta, are we certain that Hibernate cannot work with a `double` in the database? I've googled for documentation on this, but haven't found any. Could you point me to documentation stating that the list index _must_ be an `int` in the database? Have we tried making the column a `double` in the database? If we make `order_set_member.sort_weight` a `double` in the database and Hibernate is happy to use it for a list index, then I'd rather have consistency across the data model.